### ISSUE TITLE
Duplicate results for the same document

### ISSUE THREAD

**Issue by jpillora** [hashicorp/vscode-terraform#662]
### Server Version
```
serve_command.go:150: Starting terraform-ls 0.18.0
```

(from logs, not in my PATH)

### Terraform Version
```
$ terraform -version
Terraform v0.15.5
on darwin_arm64
```

### Client Version
```
vs code
Version: 1.57.0
Commit: b4c1bd0a9b03c749ea011b06c6d2676c
```

### Expected Behavior

open a workspace with 4 terraform projects, expected 1 instance of terraform-ls running

### Actual Behavior

4 instances of terraform-ls are running, all pointing at 1 of the projects:

![image](https://user-images.githubusercontent.com/633843/121630440-7f1bec00-cac0-11eb-8190-14ee1808f407.png)

with 4 duplicates of each option when i use auto-complete

<img width="618" alt="image" src="https://user-images.githubusercontent.com/633843/121630038-bf2e9f00-cabf-11eb-9541-f8e8c95e2de6.png">

### Steps to Reproduce

open a workspace with multiple terraform projects

**Comment by jpillora** [comment]
You are correct, multi-root workspaces working fine in `2.11.0`

![image](https://user-images.githubusercontent.com/633843/121671346-c8386400-caf1-11eb-89ec-1b909df07c2d.png)

**Comment by radeksimko** [comment]
I just released a patched version 2.12.1 - can you try upgrading and see if the problem is fixed? It should be, based on my testing as part of #663

**Comment by jpillora** [comment]
Sorry for the delay - fixed! Thanks :)
